sql-cli/sql_cli/__main__.py

```diff
@@ -6,28 +6,45 @@
 from rich import print as rprint
 
 import sql_cli
+from sql_cli.astro.command import AstroCommand
+from sql_cli.astro.group import AstroGroup
 from sql_cli.constants import DEFAULT_AIRFLOW_HOME, DEFAULT_DAGS_FOLDER
 
 load_dotenv()
-app = typer.Typer(add_completion=False, context_settings={"help_option_names": ["-h", "--help"]})
+app = typer.Typer(
+    name="flow",
```

Comment:
This is useful for testing the help output as it contains the program name which if not set is `root`.